File: angular-blog/src/app/models/article.ts
import { Blog } from "./blog";
import { Comment } from "./comment";
import { Like } from "./like";

export class Article{
    id?:number;
    title?:string;
    description?:string;
    photo?:string;
    idBlog?:number;
    blog?:Blog;
    likes?: Like[];
    comments?: Comment[];
}

File: BlogWebAPI/BlogWebAPI/Entities/Article.cs
namespace BlogWebAPI.Entities
{
    public class Article
    {
        public int Id { get; set; }
        public string Title { get; set; } 
        public string Description { get; set; } 
        public string? Photo { get; set; }
        public int IdBlog { get; set; }
        public Blog Blog { get; set; } 
        public ICollection<Like> Likes { get; set; }
        public ICollection<Comment> Comments { get; set; }



    }
}
